### DISCUSSION TITLE
TypeError: Illegal invocation

### DISCUSSION THREAD

**Discussion by Serega007RU** [tsukasa/BdBrowser#21] [Problems and issues]
BdBrowser stopped working, when you start the discord in the console, it gives the following error:
![image](https://user-images.githubusercontent.com/64780091/227697992-160d620c-f9ca-410b-9d93-8c400049e67b.png)

Versions:
Stable 183813 (7a19dec)
BDBrowser 1.8.5.20230309
betterdiscord.asar (v1.9.0, VFS)

**Comment by tsukasa** [comment]
Greetings,

thank you for the report! This issue is caused by an internal change on Discord's end and has been addressed through commit 0586024, which updates the webpack.js with a fixed copy from BetterDiscord.

Please note that the same problem is also present in BetterDiscord 1.9.0, so you will [either need to grab an updated copy from BetterDiscord's CI](https://github.com/BetterDiscord/BetterDiscord/actions/runs/4513336320) or build/provide your own BetterDiscord asar or renderer. I expect an official BetterDiscord update to address this issue soon.

Once 1.9.1 (or whatever the hotfix will be called) will be available, simply removing the betterdiscord.asar from the VFS and reloading should suffice to cause BdBrowser to re-download the fixed version. Instructions for pushing your own betterdiscord.asar into the VFS [can be found in the readme.md file](https://github.com/tsukasa/BdBrowser#creating-backup-files).
